tests/postgres/testing.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package postgres
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-dbw"
+	"github.com/stretchr/testify/require"
+)
+
+type user struct {
+	ID        uint
+	Name      string
+	Email     *string
+	Age       uint8
+	Birthday  *time.Time
+	CreatedAt time.Time
+}
+
+func testInsertUser(t *testing.T, rw dbw.Writer, u *user) {
+	t.Helper()
+	testCtx := context.Background()
+	require.NoError(t, rw.Create(testCtx, u))
+}
+
+func testDeleteUsers(t *testing.T, rw dbw.Writer) {
+	t.Helper()
+	testCtx := context.Background()
+	_, err := rw.Exec(testCtx, "delete from users", nil)
+	require.NoError(t, err)
+}
```

Comment:
There are no tests for this

Reply:
It's used in the tests which would fail if it wasn't working.

Reply:
I don't see anything that calls it

Reply:
And the coverage report doesn't show it being called

Reply:
I'll remove it. Ty!